release-calendar/README.md

```diff
@@ -7,4 +7,8 @@ To start: `cd release-calendar && npm install`
 To build the app: `npm run build`
 To run the app: `npm run start`
 To run tests: `npm run test`
-To lint: `npm run lint`
\ No newline at end of file
+To lint: `npm run lint`
+
+#### Local development
+1. Install MySQL for a local database.
```

Comment:
Note that we use PostgreSQL in our other apps, so it might be worth switching to that as the default db engine for consistency

Reply:
I considered it, but MySQL seemed more supported and popular. We won't notice the difference too much but I am using TypeORM instead of knex, which looks very different.